Modules/GameOptionsSender/PlayerGameOptionsSender.cs

```diff
@@ -78,8 +78,9 @@ public override IGameOptions BuildGameOptions()
         if (GameStates.IsNormalGame) AURoleOptions.SetOpt(opt);
         else if (GameStates.IsHideNSeek) return opt;
 
+        var CheckStartGameCREW = player.GetCustomRole().IsDesyncRole() && player.GetCustomRole().IsCrewmate() && !Main.introDestroyed;
         var state = Main.PlayerStates[player.PlayerId];
-        opt.BlackOut(state.IsBlackOut);
+        opt.BlackOut(state.IsBlackOut || CheckStartGameCREW);
```

Comment:
I think it's no needed anymore

Reply:
tru,
I added this when host was delayed because I sent the disconnect 2 billion times 💀but now I much imporved roletype assign system